package_managers.py

```diff
@@ -189,6 +189,21 @@ def update_db(
             num_php_http_client_downloads,
             num_node_http_client_downloads
             ):
+        parms = [
+            num_total_csharp_downloads,
+            num_nodejs_monthly_downloads,
+            num_php_downloads,
+            num_python_downloads,
+            num_ruby_downloads,
+            num_python_http_client_downloads,
+            num_python_open_source_library_data_collector_downloads,
+            num_ruby_http_client_downloads,
+            num_csharp_http_client_downloads,
+            num_php_http_client_downloads,
+            num_node_http_client_downloads]
+
+        for parm in parms:
+            parm = None if parm is None else int(parm)
```

Comment:
Is python passing by reference here or is this creating a new parameter called `parm` ?